### PR TITLE
hypershift: enable external-dns on public-private

### PR DESCRIPTION
**Pull request by LiangquanLi930**
enable external-dns

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-public-private-mgmt-p1-f7

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/hypershift/public-private/provision/cucushift-installer-rehearse-aws-ipi-ovn-hypershift-public-private-provision-chain.yaml [hunk 1]
```diff
@@ -5,8 +5,10 @@ chain:
   - chain: cucushift-hypershift-extended-install-private
   - chain: hypershift-aws-create
   env:
+  - name: EXTRA_ARGS
+    default: "--external-dns-domain=hypershift-ext.qe.devcluster.openshift.com --zones=us-east-1a,us-east-1b,us-east-1c"
```
Comment (heliubj18) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/hypershift/public-private/provision/cucushift-installer-rehearse-aws-ipi-ovn-hypershift-public-private-provision-chain.yaml:9]: better to use an ENV and set the ENV as the default value

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-public-private-mgmt-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-public-private-mgmt-p1-f7

**Comment by LiangquanLi930** [comment]
test job: https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/41140/rehearse-41140-periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-public-private-mgmt-p1-f7/1679004961188352000

**Comment by LiangquanLi930** [comment]
https://issues.redhat.com/browse/OCPQE-15504

**Pull request merged**